website/docs/components/copy/button/partials/code/composition.md

```diff
@@ -0,0 +1,49 @@
+## Composition with form inputs
```

Comment:
Creating another section under ComponentAPI isn't a pattern we use elsewhere on the docs website. It also feels relevant enough to be listed as a child of the "How to use this component" section. What do you think about nesting it there instead?

Reply:
+1: this belongs to the "how to use" section

Reply:
I initially had it under "How to use", but moved it to a new section because it seemed to be leaning a bit more towards a separate pattern (not being used in isolation). I've moved this back under "How to use" now.